clients/python/build/setup.py

```diff
@@ -0,0 +1,78 @@
+# coding: utf-8
+
+"""
+    Unity Catalog API
+
+    Official Python SDK for Unity Catalog
+"""
+
+from setuptools import setup, find_packages
+from pathlib import Path
+
+# Read the long description from README.md
+this_directory = Path(__file__).parent
+long_description = (this_directory / "README.md").read_text()
+
+setup(
+    name="unitycatalog-client",
+    version="0.2.1",
```

Comment:
pick the version from sbt 

https://github.com/delta-io/delta/blob/master/setup.py#L10-L17